components/date-input/src/DateInput.tsx

```diff
@@ -44,19 +44,24 @@ DateInput.format = (v: string): string => {
   const isSet = (v: any): boolean => (
     !!(v || v === 0)
   );
-  const [ day, month, year ] = (
```

Comment:
as above - Part of another (unapproved) PR. I don't think this should be included.